### DISCUSSION TITLE
Am I allowed to tell others?

### DISCUSSION THREAD

**Discussion by Scwestbrook** [ChAoSUnItY/EkiLib#8] [General]
Can I give out CurseForge links and Github links if I always state that the mod was made by ChAoSUnItY on Github? I want to spread awareness about this mod because it took complex item processing and put it into one or two blocks.

**Comment by ChAoSUnItY** [comment]
Of course you can! There's no limit to share this mod to others.

**Comment by Scwestbrook** [comment]
Great!
